tests/ApiApprovalTests/ApiApprovalTests.csproj

```diff
@@ -2,7 +2,7 @@
   <Import Project="../../Tests.props" />
 
   <PropertyGroup>
-    <TargetFramework>net6</TargetFramework>
+    <TargetFramework>net7</TargetFramework>
```

Comment:
net7 here and net7.0 in some other places. Let's use net7 in all projects.

Reply:
Using `net7.0` as this is the standard and, when using `net7`, VS adds new tests to unusable location in test browser

- https://learn.microsoft.com/en-us/dotnet/core/tools/dotnet-new-sdk-templates
- https://learn.microsoft.com/en-us/dotnet/standard/frameworks

Reply:
> VS adds new tests to unusable location in test browser

Could you share a screenshot?

Reply:
I think that I'm fine switching to `net7.0` instead of `net7` although I am used to the second.

Reply:
![image](https://user-images.githubusercontent.com/6377684/202380496-4e89a19b-bad3-417f-aa4a-6af71b749512.png)

Reply:
Attempting to run the new test from the test browser will fail because when VS compiles the code, it will appear under the "net7.0" tfm and so will not run.

With the targetframework set to `net7.0`, the new test is added to the proper section in the first place, and so it can be selected and run immediately.

Reply:
This is not new; it has done this for a long time.  I've just never bothered to fix it.

Reply:
And after rebuild everything becomes as it should?

Reply:
yes

Reply:
So long as we use the recommended TFMs (`net7.0`, etc) then it works in the first place.